blocks_format.md

```diff
@@ -203,20 +203,16 @@ This block can only add members, not remove them. The list of added members is t
 | encrypted_group_private_encryption_keys_for_provisional_users | list([PendingGroupEncryptedKey3](#pendinggroupencryptedkey3)) | The new group keys encrypted for the new provisional users                                     |
 | self_signature_with_current_key                               | fixed buffer (64 bytes)                                       | The signature of all non-signature fields, in that order, with the current group signature key |
 
-### UserGroupUpdate
+### UserGroupRemoval
+
+This block is not stored by the server.
```

Comment:
It is not stored in block form, however the information is kept by the server. Maybe, "is not stored in the Trustchain" ?